pypsi/commands/include.py

```diff
@@ -65,28 +67,53 @@ def run(self, shell, args):
 
     def include_file(self, shell, path):
         fp = None
-        ifile = IncludeFile(path)
+        self.ifile = IncludeFile(path)
```

Comment:
`ifile` keeps track of the line number, I made it an instance var so I could track what line I was on in the `get_next_line` function.